### ISSUE TITLE
On partition resizing, no information about required size of Linux partition

### ISSUE THREAD

**Issue by djc** [AsahiLinux/asahi-installer#31]
All of this seems to be about the target new size of the macOS partition:

<img width="464" alt="Screenshot 2022-03-18 at 19 12 30" src="https://user-images.githubusercontent.com/158471/159102849-49379fd5-7815-4284-82da-d7f8f4cd0425.png">

It would be helpful to have some clues about a desirable/minimum size for the new Linux partition, as well.

**Comment by discentem** [comment]
For anyone who wants to take a stab at this (I am thinking of giving it shot myself!) the change would be made somewhere here: https://github.com/AsahiLinux/asahi-installer/blob/main/src/main.py#L557-L564

It could be interesting to a `max` option to `We're going to resize this partition` as well, which would automatically shrink macOS _by 53gb_ (?) per the [system requirements](https://github.com/AsahiLinux/AsahiLinux.github.io/blob/main/content/blog/2022/03/17-asahi-linux-alpha-release.md#system-requirements).

**Comment by leifliddy** [comment]
I don't think this is a reasonable request as the minimum space required is completely dependent upon what your requirements are. What you plan to install, how you intend the use the system...etc. The absolute bare minimum space requirements are the size of the initial Linux + EFI partitions created, which are listed here:

https://github.com/AsahiLinux/asahi-installer/blob/main/data/installer_data.json

ie
**Minimal install**: 5GB for root and 500 MB for EFI == 5.5GB
Although probably better to bump it to 6GB to be safe.

**Comment by stevenae** [comment]
I made some changes here -- https://github.com/AsahiLinux/asahi-installer/compare/main...stevenae:asahi-installer:patch-1

Would this work?
